ddl/ddl_tiflash_api.go

```diff
@@ -463,6 +512,126 @@ func (d *ddl) pollTiFlashReplicaStatus(ctx sessionctx.Context, pollTiFlashContex
 	return allReplicaReady, nil
 }
 
+func (d *ddl) pollTiFlashPeerInfo(ctx sessionctx.Context, pollTiFlashContext *TiFlashManagementContext) (bool, error) {
+	allReplicaReady := true
+
+	if len(pollTiFlashContext.TiFlashStores) == 0 {
+		return true, nil
+	}
+
+	// Start to process every table.
+	schema := d.GetInfoSchemaWithInterceptor(ctx)
+	if schema == nil {
+		return false, errors.New("Schema is nil")
+	}
+
+	var tableMap = make(map[int64]TiFlashReplicaStatus)
+	// Collect TiFlash Replica info, for every table.
+	for _, db := range schema.AllSchemas() {
+		tbls := schema.SchemaTables(db.Name)
+		for _, tbl := range tbls {
+			tblInfo := tbl.Meta()
+			GetTiFlashReplicaMapInfo(tblInfo, &tableMap)
+		}
+	}
+
+	tiflashStoreIDs := make([]int64, len(pollTiFlashContext.TiFlashStores))
+
+	for i, store := range pollTiFlashContext.TiFlashStores {
+		tiflashStoreIDs[i-1] = store.Store.ID
+	}
+
+	sort.Slice(tiflashStoreIDs, func(i, j int) bool { return tiflashStoreIDs[i] < tiflashStoreIDs[j] })
+
+	isTiflashPeer := func(tiflashPeerIDs []int64, peerID int64) bool {
```

Comment:
PeerID and StoreID are different IDs in PD. Maybe you should correct your naming to store ID.